test/esinstall/package-entrypoints/package-entrypoints-browser.test.js

```diff
@@ -0,0 +1,66 @@
+const {install} = require('../../../esinstall/lib');
+
+/**
+ * Browser configuration
+ * https://github.com/defunctzombie/package-browser-field-spec
+ */
+describe('package-entrypoints browser configuration', () => {
+  it('is able to resolve browser: path configuration', async () => {
+    const cwd = __dirname;
+
+    const targets = [
+      // "browser": "index.js"
+      'browser-path',
+    ];
+
+    const {
+      importMap: {imports},
+    } = await install(targets, {
```

Comment:
Running multiple `install()` commands in the same test file in parallel may mean that they overlap and possibly overwrite the same files. To prevent this we could adjust the `dest` option in each, e.g. `install(targets, { dest: someLoc })`. It looks like you’re only testing the import map so files don‘t matter here; just for future cases.

Reply:
Ah yeah looks like that‘s causing issues in the test runner. Maybe try this:

```diff
const dest = path.join(cwd, 'test-browser')
  await install(targets, {
+   dest,
    cwd,
  });
```

You can then add `test-*` to a local `.gitignore` so all the build artifacts aren‘t saved in Git

Reply:
Thanks! This one had me scratching my head.

Reply:
This didn't solve it unfortunately :(. I'm unable to recreate locally on any version of Node. Will keep digging.

Reply:
I’m a bit stumped, too. I also was getting a failure locally before, but it‘s passing now. I’m not sure why.

The only thing that I’d be curious to investigate is:

```js
    expect(run).rejects.toThrow();
```

I wonder if there‘s some scenario where that doesn‘t finish, or Jest swallows the error, or something. e.g. does it work in CI without the `.rejects`? I’m completely guessing; just something for tomorrow